src/develop/develop.c

```diff
@@ -771,9 +771,9 @@ void dt_dev_second_window_configure(dt_develop_t *dev, int wd, int ht)
   int32_t tb = 0;
   if(dev->iso_12646.enabled)
   {
-    const int bsize = darktable.gui->dpi / 2.54 * dt_conf_get_float("darkroom/ui/iso12464_border");
+    const int bsize = dev->second_window.dpi * dt_conf_get_float("darkroom/ui/iso12464_border") / 2.54;
```

Comment:
To avoid mixing double/float : 2.54f

Reply:
Was not sure as the dpi is a double ??

Reply:
Ah its a double... so indeed that's not an issue.